demo/src/components/blog.js

```diff
@@ -0,0 +1,39 @@
+/* global window */
+import React, {Component, PropTypes} from 'react';
+import autobind from 'autobind-decorator';
+
+export default class Blog extends Component {
+
+  componentWillReceiveProps(nextProps) {
+    const {route} = nextProps;
+    if (this.props.route !== route) {
+    }
+  }
+
+  // redirect links to parent window
+  @autobind _onPostLoaded(evt) {
+    const doc = evt.target.contentDocument;
```

Comment:
Would be great with more explanation here. If this is building on some github functionality, a link to the relevant docs. Can't say I understand how this works.